code/modules/research/designs/autolathe_designs.dm

```diff
@@ -543,15 +543,6 @@
 	category = list("initial", "Medical", "Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SERVICE
 
-/datum/design/healthanalyzer
```

Comment:
Stealh change, the health analizer is no longer obtainable via autolathe, opt to replace it with the health sensor (like for medbots)

Reply:
Nevermind, the health sensor is there, just add the CL for the health analizer